src/main/java/org/sonatype/repository/helm/internal/metadata/IndexYamlAbsoluteUrlRewriter.java

```diff
@@ -108,10 +108,10 @@ private Event maybeSetAbsoluteUrlAsRelative(ScalarEvent scalarEvent) throws Malf
       URI uri = new URIBuilder(oldUrl).build();
       if (uri.isAbsolute()) {
         String fileName = uri.getPath();
-        // Strip leading slash
-        if (!fileName.isEmpty()) {
-            fileName = fileName.substring(1);
-        }
+        //remove leading "/" to avoid URI missing, by ref: https://github.com/helm/helm/issues/3878
```

Comment:
Pretty sure this is not necessary, the last change should do the same thing.

Reply:
The modification of this place is not the same as before, here will solve the problem of a secondary directory, like this chart repo address

https://burdenbear.github.io/kube-charts-mirror

https://kubernetes.oss-cn-hangzhou.aliyuncs.com/charts

Reply:
Didn't catch that at first! Could you make a different PR for this change? We try and keep PRs isolated to one feature if possible, so it's more apparent in the merge what was done.